libs/custom-paper-handler/src/driver/driver.test.ts

```diff
@@ -39,6 +39,12 @@ let mockWebUsbDevice: MockWebUsbDevice;
 let paperHandlerWebDevice: WebUSBDevice;
 let paperHandlerDriver: PaperHandlerDriver;
 
+function arrayBufferOf(...numbers: number[]): ArrayBuffer {
```

Comment:
Should we put something like this in libs/basics for all the cases you needed to cover in this PR?

Reply:
I could see a case for `@votingworks/test-utils`. WDYT?

Reply:
I guess I was thinking of the pattern used in the previous commits in this PR:

```ts
   const arrayBuffer = new ArrayBuffer(data.byteLength);
    const arrayBufferView = new Uint8Array(arrayBuffer);
    data.copy(arrayBufferView);
```

which is slightly different from this use case. I think some of those cases are not in test files

Reply:
Added `arrayBufferFrom` to `libs/basics`.